test/cog/queries/command_test.exs

```diff
@@ -7,7 +7,7 @@ defmodule Cog.Queries.Command.Test do
   alias Cog.Repo
 
   setup do
-    {:ok, bundle} = Repo.insert(%Bundle{name: "test_bundle", config_file: %{}, manifest_file: %{}})
+    {:ok, bundle} = Repo.insert(%Bundle{name: "test_bundle", config_file: %{}})
```

Comment:
I think `Repo.insert!` might give us a better error message here instead of a bad match against `{:ok, bundle}`.